docs/using-az-aro.md

```diff
@@ -140,6 +140,7 @@ cluster:
      --vnet dev-vnet \
      --master-subnet "$CLUSTER-master" \
      --worker-subnet "$CLUSTER-worker"
+     --pull-secret $PULL_SECRET
```

Comment:
I think the step of setting `$PULL_SECRET` is missing from this doc.

Reply:
Yes, its part of our `./env`. Will update